protocol/contracts/libraries/Silo/LibSilo.sol

```diff
@@ -228,42 +228,13 @@ library LibSilo {
                 .add(stalk);
         }
 
-        // emit event.
-        emit LibGerminate.FarmerGerminatingStalkBalanceChanged(
-            account,
-            stalk
-        );
+        // emit events.
+        emit LibGerminate.FarmerGerminatingStalkBalanceChanged(account, stalk);
+        emit LibGerminate.TotalGerminatingStalkChanged(season, stalk);
```

Comment:
same here, believe It should be the germinating season

Reply:
fixed here: https://github.com/BeanstalkFarms/Beanstalk/pull/837/commits/cdc5a0353e9298e5d74888fde498d3a9988114cc#diff-fbd431ce899b0679e919b2bf1fa828e50eeddd381e4efd2012eca396bb7758daR235